components/automate-ui/src/app/pages/project/list/project-list.component.ts

```diff
@@ -103,6 +103,42 @@ export class ProjectListComponent implements OnInit, OnDestroy {
     this.layoutFacade.showSettingsSidebar();
     this.projects.getApplyRulesStatus();
     this.store.dispatch(new GetProjects());
+
+    // handle project creation success response
+    this.store.pipe(
+      select(createStatus),
+      takeUntil(this.isDestroyed),
+      filter(state => {
+        return this.creatingProject && !pending(state);
+      }))
+      .subscribe(state => {
+        if (state === EntityStatus.loadingSuccess) {
```

Comment:
We could add this to the filter. It would be `this.creatingProject && !pending(state) && state === EntityStatus.loadingSuccess`